apache-hbase-mixin/config.libsonnet

```diff
@@ -1,9 +1,9 @@
 {
   local this = self,
-  filteringSelector: 'job="integrations/apache-hbase"',
+  filteringSelector: 'job=~".+"',  // set to apply static filters to all queries and alerts, i.e. job="bar"
```

Comment:
If you pull latest logs-lib into the vendor, you should be able to get away with just an empty string here, but this is fine too :)